src/Avalonia.Controls/TopLevel.cs

```diff
@@ -232,6 +235,18 @@ protected bool AutoSizing
         }
 
         /// <inheritdoc/>
+        IRenderTarget IRenderRoot.CreateRenderTarget()
+        {
+            return _renderInterface.CreateRenderer(PlatformImpl.Handle);
```

Comment:
Well, `IRenderTarget` *is* useful for tracking state from the rendering system (direct2d/opengl context, offscreen surface, etc), but lifetime management of said render target should be the job of `ITopLevelImpl`, I think.